src/App.tsx

```diff
@@ -1,14 +1,23 @@
 import { ThemeProvider as MuiThemeProvider } from '@material-ui/core';
 import React from 'react';
+import { StylesProvider } from '@material-ui/core/styles';
 import { ThemeProvider } from 'styled-components';
 import { DemoContainer } from './demo/DemoContainer';
 import { theme } from './layout/theme';
+import { CategoryCard } from './CategoryCard';
 
 export const App: React.FC<{}> = () => (
-  <MuiThemeProvider theme={theme}>
-    <ThemeProvider theme={theme}>
-      {/* Replace with actual content */}
-      <DemoContainer />
-    </ThemeProvider>
-  </MuiThemeProvider>
+  <StylesProvider injectFirst>
+    <MuiThemeProvider theme={theme}>
+      <ThemeProvider theme={theme}>
+        {/* Replace with actual content */}
+        <DemoContainer />
+        <CategoryCard
```

Comment:
I've added this only to show that it works. It probably should be moved to the layout page. I can remove it from this PR if needed.

Reply:
I think you can remove it, beacuse we don't need it for the moment, but it looks nice ;)

Reply:
Done ✅